data/helpers.d/user

```diff
@@ -116,11 +131,21 @@ ynh_system_user_delete () {
     # Manage arguments with getopts
     ynh_handle_getopts_args "$@"
 
-    if ynh_system_user_exists "$username"	# Check if the user exists on the system
+    # Check if the user exists on the system
+    if ynh_system_user_exists "$username"
     then
 		echo "Remove the user $username" >&2
-		sudo userdel $username
+		deluser $username
 	else
 		echo "The user $username was not found" >&2
     fi
+
+    # Check if the group exists on the system
+    if ynh_system_group_exists "$username"
+    then
+		echo "Remove the group $username" >&2
+		delgroup $username
+	else
+		echo "The group $username was not found" >&2
```

Comment:
Do we really want to display this warning, considering that it should be okay that the group is not found (in the nominal case, it should have been deleted with the previous `deluser`)

Reply:
Nope, `deluser` will not remove the group anymore.
`userdel` would have done it ;)

Reply:
Uhwokay x)

Reply:
From the man of `deluser`
> If called with one non-option argument and without the --group  option, deluser will remove a normal user.

deluser removes the user **and** the group...
You were right ;)

Anyway... if the group isn't empty, the issue at the origin of this PR, delgroup will be called and the group removed.
The user I added to this group wasn't removed, only the group. Everything's fine.

Reply:
:+1: